packages/core/src/modules/credentials/protocol/v2/V2CredentialService.ts

```diff
@@ -221,6 +221,7 @@ export class V2CredentialService extends CredentialService {
     const options: ServiceOfferCredentialOptions = {
       credentialFormats: proposal.credentialFormats,
       comment: proposal.comment,
+      protocolVersion: credentialRecord.protocolVersion,
     }
     const message = await this.createOfferAsResponse(credentialRecord, options)
```

Comment:
When creating an offer as response we can take the protocol version from the credential record

Reply:
This also seems unresolved @NB-MikeRichardson